vast/integration/reference/node-suricata-rrdata/step_01.ref

```diff
@@ -1,2 +1,2 @@
+<2017-05-09T21:07:44.210977, 479971984293854, 6, nil, nil, 10.16.1.11, 54380, 8.8.8.8, 53, "UDP", "dns", nil, 2, "answer", 22038, "8180", T, T, T, nil, nil, "google.com", "MX", "NOERROR", nil, nil, nil, nil, nil, nil, nil, ["aspmx.l.google.com", "alt1.aspmx.l.google.com", "alt2.aspmx.l.google.com", "alt3.aspmx.l.google.com", "alt4.aspmx.l.google.com"], nil, nil, nil, nil, [<"rrname": "google.com", "rrtype": "MX", "ttl": 599, "rdata": "aspmx.l.google.com", "soa": <"mname": nil, "rname": nil, "serial": nil, "refresh": nil, "retry": nil, "expire": nil, "minimum": nil>, "sshfp": <"fingerprint": nil, "algo": nil, "type": nil>, "srv": <"priority": nil, "weight": nil, "port": nil, "name": nil>>, <"rrname": "google.com", "rrtype": "MX", "ttl": 599, "rdata": "alt1.aspmx.l.google.com", "soa": <"mname": nil, "rname": nil, "serial": nil, "refresh": nil, "retry": nil, "expire": nil, "minimum": nil>, "sshfp": <"fingerprint": nil, "algo": nil, "type": nil>, "srv": <"priority": nil, "weight": nil, "port": nil, "name": "xxx">>, <"rrname": "google.com", "rrtype": "MX", "ttl": 599, "rdata": "alt2.aspmx.l.google.com", "soa": <"mname": "foo", "rname": nil, "serial": nil, "refresh": nil, "retry": 42, "expire": nil, "minimum": nil>, "sshfp": <"fingerprint": nil, "algo": nil, "type": nil>, "srv": <"priority": nil, "weight": nil, "port": nil, "name": nil>>, <"rrname": "google.com", "rrtype": "MX", "ttl": 599, "rdata": "alt3.aspmx.l.google.com", "soa": <"mname": nil, "rname": nil, "serial": nil, "refresh": nil, "retry": nil, "expire": nil, "minimum": nil>, "sshfp": <"fingerprint": nil, "algo": nil, "type": nil>, "srv": <"priority": nil, "weight": nil, "port": nil, "name": nil>>, <"rrname": "google.com", "rrtype": "MX", "ttl": 599, "rdata": "alt4.aspmx.l.google.com", "soa": <"mname": nil, "rname": nil, "serial": nil, "refresh": nil, "retry": nil, "expire": nil, "minimum": nil>, "sshfp": <"fingerprint": nil, "algo": nil, "type": nil>, "srv": <"priority": nil, "weight": nil, "port": nil, "name": nil>>]>
```

Comment:
From peer-review with @tobim: We want to add a workaround to the JSON reader to revert to the old behavior, even though the new one is more consistent. In the future we want to stop flattening entirely, but this requires a redesign of the table slice builder API.

Reply:
This is now implemented.